### PR TITLE
Remove CodepointMappingProto from ClientState.

### PR DESCRIPTION
**Pull request by andyjgf**

### CODE REVIEW

File: patch_subset/codepoint_map.cc [hunk 1]
```diff
@@ -28,7 +28,19 @@ void CodepointMap::FromProto(const CodepointRemappingProto& proto) {
...
+  int index = 0;
+  int last_cp = 0;
+  for (int delta : proto.deltas()) {
+    int cp = last_cp + delta;
+    AddMapping(cp, index++);
+    last_cp = cp;
+  }
+}
+
+StatusCode CodepointMap::ToProtoOld(CodepointRemappingProto* proto) const {
```
Comment (garretrieger) [inline patch_subset/codepoint_map.cc:43]: and this one.

File: patch_subset/patch_subset_client_test.cc [hunk 2]
```diff
@@ -70,7 +70,6 @@ class PatchSubsetClientTest : public ::testing::Test {
   PatchResponseProto CreateResponse(ResponseType type) {
     PatchResponseProto response;
     response.set_original_font_fingerprint(kOriginalFingerprint);
-    response.set_format(PatchFormat::BROTLI_SHARED_DICT);
```
Comment (garretrieger) [inline patch_subset/patch_subset_client_test.cc:73]: Was this intentionally removed?

**Review by garretrieger — APPROVED** [review]

File: patch_subset/codepoint_map.cc [hunk 3]
```diff
@@ -16,7 +16,7 @@ void CodepointMap::AddMapping(hb_codepoint_t from, hb_codepoint_t to) {
   decode_map[to] = from;
 }
 
-void CodepointMap::FromProto(const CodepointRemappingProto& proto) {
+void CodepointMap::FromProtoOld(const CodepointRemappingProto& proto) {
```
[the remark this answers is not in the document]
Comment (andyjgf) [reply patch_subset/codepoint_map.cc:19]: Not yet. Next PR.

File: patch_subset/codepoint_map.cc (hunk 1 above)
Comment (andyjgf) [reply patch_subset/codepoint_map.cc:43]: Next one.

File: patch_subset/patch_subset_client_test.cc (hunk 2 above)
Comment (andyjgf) [reply patch_subset/patch_subset_client_test.cc]: It can stay.

**Pull request merged**